concent_api/core/validation.py

```diff
@@ -106,6 +106,23 @@ def validate_list_of_identical_task_to_compute(list_of_task_to_compute: List[mes
     return True
 
 
+def validate_list_of_identical_report_computed_task(list_of_report_computed_task: List[message.tasks.ReportComputedTask]):
+    assert isinstance(list_of_report_computed_task, list)
+    assert all([isinstance(report_computed_task, message.tasks.ReportComputedTask) for report_computed_task in list_of_report_computed_task])
+
+    if len(list_of_report_computed_task) <= 1:
+        return False
+
+    base_report_computed_task = list_of_report_computed_task[0]
+
+    for report_computed_task in list_of_report_computed_task:
+        for slot in message.tasks.ReportComputedTask.__slots__:
+            if getattr(base_report_computed_task, slot) != getattr(report_computed_task, slot):
```

Comment:
Doesn't Golem's `Message` have an equality operator? If not, maybe it would make sense to add it there and make a pull request?

Reply:
OK. Makes sense.